scripts/ripe-atlas

```diff
@@ -1,94 +1,10 @@
 #!/usr/bin/env python
-
-import importlib
-import os
-import pkgutil
-import re
 import sys
 
-from ripe.atlas.tools import commands
+from ripe.atlas.tools.script import RipeAtlas
```

Comment:
I've checked and it isn't required for this change, so if there are objections then I can leave it as it is.